muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/records/body/application/matriculation-eligibility-row/matriculation-eligibility-row.tsx

```diff
@@ -0,0 +1,174 @@
+import * as React from "react";
+import { connect } from "react-redux";
+import { i18nType } from "~/reducers/base/i18n";
+import { Dispatch } from "redux";
+import { StateType } from '~/reducers';
+import mApi from '~/lib/mApi';
+
+/**
+ * Enum describing matriculation eligibility
+ * 
+ * @author Antti Leppä <antti.leppa@metatavu.fi>
+ */
+enum EligbleEnum { 
+  TRUE, 
```

Comment:
So TRUE is 0, and FALSE is 1

Is that intented?

Reply:
I thought that in enum it does not really matter, but i'll change the order into FALSE, TRUE, UNKNOWN

Reply:
Order changed